subscribe curator cluster to container storage operator

larsks opened the pull request:
we are going to try getting the curator cluster connected to an
external ceph cluster for storage.

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **humairak** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @humairak` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fapps).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/apps/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["humairak"]} -->

tumido reviewed (changes requested):
You need to include the `OperatorGroup` as well, otherwise the subscription wouldn't be picked up and operator won't install.

See the zero spec:
https://github.com/operate-first/apps/blob/2c27ba83acfe957037a1b967139d3d6238491d10/cluster-scope/overlays/moc/zero/kustomization.yaml#L102

larsks commented:
@tumido thanks, I added the operatorgroup, and as long as I was making changes I added the openshift-storage namespace as well, which we're going to need as soon as we get around to creating a storagecluster (that will probably happen manually first, as we work out exactly what we need to do for external ceph connectivity).

tumido reviewed (commented):
/lgtm

sesheta commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/operate-first/apps/pull/449#pullrequestreview-630287768" title="Approved">HumairAK</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fapps).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/operate-first/apps/blob/master/OWNERS)~~ [HumairAK]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->
